nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/avro/Constants.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+package org.nuxeo.ecm.core.io.avro;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Avro local constants.
```

Comment:
"Constants" is too vague. Please explain the role of the constants, where they are used. Are these all Avro field names?